content/en/docs/v3.5/quickstart.md

```diff
@@ -1,61 +1,120 @@
 
-    ```console
-    $ etcd
-    {"level":"info","ts":"2021-09-17T09:19:32.783-0400","caller":"etcdmain/etcd.go:72","msg":... }
-    ⋮
-    ```
+## Steps
 
-    {{% alert color="info" %}}**Note**: The output produced by `etcd` are
-    [logs](../op-guide/configuration/#logging) &mdash; info-level logs can
-    be ignored. {{% /alert %}}
+### 1. Installation
 
- 3. From **another terminal**, use `etcdctl` to set a key:
+Install `etcd` from pre-built binaries using cURL.
 
-    ```console
-    $ etcdctl put greeting "Hello, etcd"
-    OK
-    ```
+   ```bash
+   # Download and extract the latest release using cURL
+   curl -L https://github.com/etcd-io/etcd/releases/download/v3.5.21/etcd-v3.5.21-linux-arm64.tar.gz \
+   -o etcd.tar.gz
+   tar -xvf etcd.tar.gz
+   cd etcd-v3.5.21-linux-arm64
 
- 4. From the same terminal, retrieve the key:
+   ```
 
-    ```console
-    $ etcdctl get greeting
-    greeting
-    Hello, etcd
-    ```
+   Move binaries to `/usr/local/bin`:
+
+   ```bash
+   sudo mv etcd etcdctl etcdutl /usr/local/bin
+   ```
+
+   Confirm installation:
+
+   ```bash
+   etcd --version
+   etcdctl version
+   ```
+
+   Output:
+
+   ```bash
+   etcd Version: 3.5.21
+   Git SHA: a17edfd
+   Go Version: go1.23.7
+   Go OS/Arch: linux/arm64
+
+
+   etcdctl version: 3.5.21
+   API version: 3.5
+
+   ```
+
+### 2. Start `etcd`
+
+   In a new terminal window, start etcd with the default configuration:
+
+   ```bash
+    etcd
+   ```
+
+   You should see logs indicating etcd has started and is listening on `localhost:2379`.
+
+### 3. From another terminal, use `etcdctl` to set a key
+
+   ```bash
+      etcdctl put greeting "Hello, etcd"
+   ```
+
+   where:
+
+- `put`: Puts the given key into the store
+
+   Ouput:
+
+   ```bash
+   OK
+   ```
+
+### 4. From the same terminal, retrieve the key
+
+   ```bash
+      etcdctl get greeting
+   ```
+
+   where:
+
+- `get`: gets the key or range of keys.
+
+   Output:
+
+   ```bash
+   greeting
+   Hello, etcd
+   ```
+
+### 5. Shut down `etcd`
+
+To stop etcd, press `Ctrl+C` in the terminal where it is running.
 
 ## What's next?
 
 Learn about more ways to configure and use etcd from the following pages:
 
-- Explore the gRPC [API][].
-- Set up a [multi-machine cluster][clustering].
-- Learn how to [configure][] etcd.
-- Find [language bindings and tools][integrations].
-- Use TLS to [secure an etcd cluster][security].
-- [Tune etcd][tuning].
-
-[api]: /docs/{{< param version >}}/learning/api
-[clustering]: /docs/{{< param version >}}/op-guide/clustering
-[configure]: /docs/{{< param version >}}/op-guide/configuration
-[integrations]: /docs/{{< param version >}}/integrations
-[security]: /docs/{{< param version >}}/op-guide/security
-[tuning]: /docs/{{< param version >}}/tuning
-[Install]: ../install/
+- If you are a developer:
+  - Explore the gRPC [API](/docs/v3.5/learning/api).
+  - Set up a [multi-machine cluster](/docs/v3.5/op-guide/clustering).
```

Comment:
This should probably go in the operator section, rather than the developer section.